tools/standardize_config_json.py

```diff
@@ -104,24 +103,57 @@ def print_diff(original: collections.OrderedDict, new: collections.OrderedDict):
     LOG.info("\n".join(diff))
 
 
-def process(filename: str):
-    contents = read_json_file(filename)
-    original = read_json_file(filename)
+def process(config_json_path: str, configuration: dict):
+    contents = read_json_file(config_json_path)
+    original = read_json_file(config_json_path)
 
     reordered = reorder_service(contents["properties"]["service"]["properties"])
     contents["properties"]["service"]["properties"] = reordered
 
+    sections = configuration.get("sections", {})
+    for section_name, head_and_tail in sections.items():
+        head = head_and_tail["head"]
+        tail = head_and_tail["tail"]
+        properties = contents["properties"][section_name]["properties"]
```

Comment:
This will fail if the config file gets out of sync and the file does not contain the specified section. Should we check and log an error?

Reply:
Fixed in a065501b359f31ed7741b29565283a6f234cea4e.